code/modules/events/ghost_role/paradox_clone_event.dm

```diff
@@ -0,0 +1,85 @@
+/datum/round_event_control/paradox_clone
```

Comment:
I don't know if I like this being an event on top of being a ruleset already, it's a lot of copy paste code and doesn't show up on the roundend report for threats, nor does it cost any.